tests/integration/test_integration.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from todo_app import create_app
+from unittest.mock import patch
+import json
+from bs4 import BeautifulSoup as bs
+
+
+def test_find_tests():
```

Comment:
Once the tests are working, we should remove all dev code like this

Reply:
yes, will make that change